docker/justfile

```
@@ -53,6 +53,13 @@ test: test-js test-py
 
 # run dev server in docker container
 serve env="dev" *args="":
+    #!/usr/bin/env bash
+    if [ "$env" = "prod" ]; then
+        echo foo
+        SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

```suggestion
        SCRIPT_DIR={{justfile_directory()}}
```
I think this is nicer way to locate the directory?

---

Did not know this was a thing :+1:

---

for me (on macos) the original `SCRIPT_DIR` points to `/var/folders/y8/455d21sn4zb237jyyl3q6z3c0000gn/T/justByxnRZ`, whereas @bloodearnest 's suggestion points to what I think is the expected `/Users/tom/Documents/bennett-institute/opencodelists/docker`